src/components/Navigation.tsx

```diff
@@ -9,16 +9,22 @@ function Navigation(): JSX.Element {
     <nav role="navigation">
       <ul className={styles.navigation}>
```

Comment:
The original component has this as a `ul`, but I think a `Flex` will suffice here instead